### PR TITLE
chore: more deflake

### PR DESCRIPTION
**Pull request by vsiravar**
Issue #, if available:

*Description of changes:*
- change `vm stop` -> `vm stop -f` and `vm remove` -> `vm remove -f`
*Testing done:*
Yes on macOS

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ginglis13 — APPROVED** [review]

**Review by pendo324 — APPROVED** [review]

File: e2e/vm/config_windows_test.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+//go:build windows
+
+package vm
+
+import (
+	"os"
+	"path/filepath"
+)
+
+var finchConfigFilePath = filepath.Join(os.Getenv("LOCALAPPDATA")+".finch", "finch.yaml")
```
Comment (pendo324) [inline e2e/vm/config_windows_test.go:12]: nit: wouldn't this also work?
```suggestion
var finchConfigFilePath = filepath.Join(os.Getenv("LOCALAPPDATA"), ".finch", "finch.yaml")

```

Comment (vsiravar) [reply e2e/vm/config_windows_test.go]: Updated.

File: e2e/vm/lifecycle_test.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 
 var testVMLifecycle = func(o *option.Option) {
 	// These tests are run in serial because we only define one virtual machine instance.
-	ginkgo.Describe("virtual machine lifecycle", ginkgo.Serial, func() {
+	ginkgo.FDescribe("virtual machine lifecycle", ginkgo.Serial, func() {
```
Comment (ginglis13) [inline e2e/vm/lifecycle_test.go:15]: nit
```suggestion
	ginkgo.Describe("virtual machine lifecycle", ginkgo.Serial, func() {
```

**Review by ginglis13 — APPROVED** [review]
nit otherwise LGTM

**Pull request merged**